README.md

```diff
@@ -135,6 +135,34 @@ val yaml = Color("CadetBlue", 95, 158, 160).toYaml
 val color = yaml.convertTo[Color]
 ```
 
+Example with inner objects:
+
+```scala
+case class Color(name: String, red: Int, green: Int, blue: Int)
+case class Pallete(name: String, colors: Option[List[Color]] = None)
+
+object PalletleYamlProtocol extends DefaultYamlProtocol {
+  implicit val colorFormat = yamlFormat4(Color)
+  implicit val palleteFormat = yamlFormat2(Color)
+}
+
+import PalletleYamlProtocol._
+import net.jcazevedo.moultingyaml._
```

Comment:
Since the previous example has `import`, I would keep them in this one as well. Makes the code more explicit of what it needs to work.